spec/spec_helper.rb

```diff
@@ -14,7 +14,7 @@
 
   def assert_correct_indenting(extension='rb', string)
     whitespace = string.scan(/^\s*/).first
-    string = string.split("\n").map { |line| line.gsub /^#{whitespace}/, '' }.join("\n").strip
+    string = string.split("\n").map { |line| line.gsub(/^#{whitespace}/, '') }.join("\n").strip
```

Comment:
Correction, doesn't do what `<<~` does. I think it looks at the indentation of the first line and then strips that off the rest of the lines.